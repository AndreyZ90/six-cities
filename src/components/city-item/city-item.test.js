import React from 'react';
import TestRenderer from 'react-test-renderer';

import CityItem from '@/components/city-item/city-item';

const city = `Amsterdam`;

describe(`CityItem component snapshot`, () => {
  test(`Should correctly render CityItem component (active)`, () => {
    const tree = TestRenderer.create(
        <CityItem city={city} isActive={true} onCurrentCityChange={() => {}} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test(`Should correctly render CityItem component (not active)`, () => {
    const tree = TestRenderer.create(
        <CityItem city={city} isActive={false} onCurrentCityChange={() => {}} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
