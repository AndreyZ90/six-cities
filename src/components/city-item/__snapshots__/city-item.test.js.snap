// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityItem component snapshot Should correctly render CityItem component (active) 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;

exports[`CityItem component snapshot Should correctly render CityItem component (not active) 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item "
    href="#"
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
